spec/webview-spec.ts

```diff
@@ -2106,7 +2108,8 @@ describe('<webview> tag', function () {
     });
 
     // TODO(miniak): figure out why this is failing on windows
-    ifdescribe(process.platform !== 'win32')('<webview>.capturePage()', () => {
+    // TODO(vertedinde): figure out why this is failing on mac arm64
+    ifdescribe(process.platform !== 'win32' && !isMacArm64)('<webview>.capturePage()', () => {
```

Comment:
Note: These failures aren't specific to the roll, they're also happening in main and have been since at least mid-last week. Filed an issue here to follow up: https://github.com/electron/electron/issues/41493